Documentation/network/servicemesh/l7-traffic-management.rst

```diff
@@ -45,9 +45,13 @@ Caveats
   the original source address/port. This may cause 5-tuple collisions when pods
   send multiple requests over the same pipelined HTTP/1.1 or HTTP/2 connection.
   (The Cilium agent assumes all CECs with parentRefs pointing to the Cilium
-  Ingress or Gateway API controllers have annotation
-  ``cec.cilium.io/use-original-source-address`` set to ``"false"``, but all other CECs
-  are assumed to have this annotation set to ``"true"``.)
+  Ingress or Gateway API controllers have
+  ``cilium.io/use-original-source-address`` set to ``"false"``, but all other CECs
+  are assumed to have this label set to ``"true"``.)
+*  For cases when CiliumEnvoyConfig is not used directly (Ingress, Gateway),
+   one could override default usage of the original source address/port in Envoy by
+   setting flag ``--proxy-use-original-source-address`` to ``false``.
```

Comment:
Consistency for indentation and formatting, please

```suggestion
  are assumed to have this label set to ``"true"``.)
* For cases when ``CiliumEnvoyConfig`` is not used directly (Ingress, Gateway),
  you can override the default usage of the original source address/port in Envoy by
  setting flag ``--proxy-use-original-source-address`` to ``false``.
```

What does `override default usage of the original source address/port` mean exactly? This part reads like we use the original source address/port by default, but we're trying to change (“override”) this behaviour. But reading the flag name and the associated PR, it sounds like we do try to use the original source address (instead of what default behaviour?). Can you please rephrase to better explain how the flag changes the behaviour?